### DISCUSSION TITLE
Is this still being maintained?

### DISCUSSION THREAD

**Discussion by jpwallace22** [heroui-inc/tailwind-variants#57] [General]
I've noticed the bugs building up and not any traction on them. Just checking if it its being actively maintained. Thank you!

**Comment by jrgarciadev** [comment]
Hey @jpwallace22 yes it is still maintained and tested by ourselves, I'm trying my best to release the v2 of NextUI https://github.com/nextui-org/nextui/pull/998 and fix the tv issues

**Reply by jpwallace22** [reply]
Hey no worries! Just checking. I love the idea behind the project and it aligns with my preferred patterns pretty well. I was thinking about contributing but noticed a build-up of bugs and a single stale PR, just wanted to check before giving some time. Thanks for the super fast response!
